com.hoptodesk.HopToDesk.metainfo.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.hoptodesk.HopToDesk</id>
+  
+  <name>HopToDesk</name>
+  <summary>Free Remote Desktop Software. Easily access remote devices securely with end-to-end encryption.</summary>
+
+  <developer id="com.hoptodesk">
+    <name>Begonia Holdings</name>
+  </developer>
+  
+  <developer_name>Begonia Holdings</developer_name>
+  
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
+  
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  
+  <description>
+    <p>
+      HopToDesk is a free remote desktop tool allowing users to share their screen, transfer files, and allow remote control access to their computers and devices using end-to-end encryption.&quot;
+    </p>
+  </description>
+
+  <url type="homepage">https://www.hoptodesk.com</url>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/bakabtw/hoptodesk-flatpak/main/screenshot.png</image>
```

Comment:
why is screenshot coming from your repo?

Reply:
I couldn't find direct links for screenshots neither in upstream, nor on website. I asked the author to add screenshots to their repo, so I can reference them

Reply:
URL has been updated to use a screenshot from upstream